frontend/src/services/EnvironmentSettings.ts

```diff
@@ -66,6 +67,10 @@ export class EnvironmentSettings {
         return false
     }
 
+    isProdEnvironment() {
+        return window.localStorage.getItem(this.environmentDisplayNameKey) == ''
```

Comment:
I feel use display name to check environment feels weird... in the production environment, the "target_env" value will be "prod"

Reply:
Yes, this seems to bit strange as I was thinking the same thing too. I saw previous one `isDevEnvironment` was using the same displayName and don't know why so just to keep it consistent. We can discuss this.

Reply:
This is refactored.